src/sys/signal.rs

```diff
@@ -210,6 +211,19 @@ libc_bitflags!{
     }
 }
 
+#[cfg(target_os = "android")]
```

Comment:
They're defined as `c_int` for all platforms.

Since you mentioned the upstream issue, please also add a `FIXME` that links to the upstream issue and states what can be done when it's resolved. We need to track these tasks in the code, not just here in the PRs.